index.js

```diff
@@ -285,10 +285,7 @@ function $asDatetime (date, skipQuotes) {
 function $asDate (date, skipQuotes) {
   const quotes = skipQuotes === true ? '' : '"'
   if (date instanceof Date) {
-    const year = new Intl.DateTimeFormat('en', { year: 'numeric' }).format(date)
-    const month = new Intl.DateTimeFormat('en', { month: '2-digit' }).format(date)
-    const day = new Intl.DateTimeFormat('en', { day: '2-digit' }).format(date)
-    return quotes + year + '-' + month + '-' + day + quotes
+    return quotes + new Date(date.getTime() - (date.getTimezoneOffset() * 60000 )).toISOString().slice(0, 10) + quotes
```

Comment:
Would you mind to add a test for this?

```
console.log(s(new Date('August 19, 1975 00:15:30 GMT+01:00'))) // "1975-08-19"
```

I tried a bit and it is aligned with the old code output

Reply:
Added a test for that specific case

Reply:
Missing git push?

Reply:
😅